tests/test_runner.py

```diff
@@ -0,0 +1,28 @@
+from unittest.mock import Mock, call
+
+import pytest
+
+from pytest_django.runner import PytestTestRunner
+
+
+@pytest.mark.parametrize(
+    *(
```

Comment:
Can you remove the `*(`? (It seems redundant)

Reply:
NM... looks like its happy..